packages/util/src/array.ts

```diff
@@ -49,3 +49,29 @@ export function splitInChunks<T>(arr: T[], chunkSize: number): T[][] {
   }
   return result;
 }
+
+/**
+ * We want to provide methods which accept a variable single number of elements and arrays.
+ * The overloaded signature to achieve this is:
+ * function doSomething(array:T[])
+ * function doSomething(...varArgs:T[])
+ * functiondoSomething(first:undefined|T|T[],...rest:T[]){
+ *   //implementation
+ * }
+ * This wrapper methods makes it easy build an array from the input.
+ * @param firstOrArray - Either an array, the first element of the var args or undefined if no argument was given.
+ * @param rest - Second to last element if var args were used, empty array if the frist argument is an array.
+ * @returns Array from the input or empty array if no input was given.
+ */
+export function variableArgumentToArray<T>(
```

Comment:
Shouldn't it be `variadic` not `variable`?

Reply:
?

Reply:
https://en.wikipedia.org/wiki/Variadic_function#:~:text=In%20mathematics%20and%20in%20computer,dating%20back%20to%201936%2D1937.

Reply:
was not aware of this word. Hm I am not sure is this common knowledge?

Reply:
I only know it under this word (varargs) and rest parameters.

Reply:
but is is rather speaking so I renamed.